jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -111,9 +111,13 @@ def activate(self):
     def deactivate(self):
         self.viewer.remove_event_callback(self.on_click)
 
-    def on_click(self, *args):
+    def on_click(self, data):
```

Comment:
If you really want to test this, you can write a test that is not unlike that is being done in https://github.com/spacetelescope/jdaviz/blob/main/jdaviz/configs/imviz/tests/test_linking.py 😉

Reply:
Since that test already uses the "b" key twice, I just changed one of them to make use of the click (so that we have coverage for both cases).